GEM/config_files/continuous_monitoring/continuous_monitoring_end2end.json

```diff
@@ -0,0 +1,7 @@
+[
+  {"**catalog_update": "${config_path}/01_catalog.json"},
+  {"**incremental_download": "${config_path}/02-incremental_download.json"},
+  {"**esa_worldcover": "${config_path}/03-download_ESA_worldcover.json"},
```

Comment:
I think all of these need to be renamed to align them with the new config filenames.